.gitlab/package_build/dmg.yml

```diff
@@ -23,7 +23,9 @@
 agent_dmg-x64-a7:
   extends: .agent_build_common_dmg
   rules:
-    - !reference [.on_a7]
+    - !reference [.on_omnibus_change]
```

Comment:
I think we should update the "on omnibus change" rule, it is too restrictive (it only triggers on changes to `omnibus/`). For instance, it doesn't trigger on Gitlab CI config changes, which do affect packaging.

I think this should be renamed to `.on_packaging_changes`, and at least include changes to `.gitlab-ci.yml`, `.gitlab/package_build.yml`, `release.json`, and files in `.gitlab/package_build`.

Reply:
Good point, on it.

Reply:
Fixed with [e91d0b0](https://github.com/DataDog/datadog-agent/pull/22882/commits/e91d0b0f2b1758cf1fa59031f9133cb033de9205) !